tests_and_analysis/test/script_tests/test_dos.py

```diff
@@ -23,6 +24,8 @@
     pass
 
 nah_phonon_file = get_castep_path('NaH', 'NaH.phonon')
+nacl_no_evec_yaml_file = os.path.join(
+    get_phonopy_path('NaCl', 'mesh'), 'mesh_no_evec.yaml')
```

Comment:
### Not a request for this PR

Would be nice if these *get_path* functions returned Path objects, then these `join` calls could become e.g.

```python
nacl_no_evec_yaml_file = get_phonopy_path('NaCl', 'mesh') / 'mesh_no_evec.yaml'
```

Reply:
Or we should just let `get_phonopy_path` take multiple filenames? That would be closer to what it's supposed to do already.

```python
def get_phonopy_path(material: str, *filename: Tuple[str]) -> str:
    return os.path.join(get_data_path(), 'phonopy_files', material, *filename)
    
nacl_no_evec_yaml_file = get_phonopy_path('NaCl', 'mesh', 'mesh_no_evec.yaml')
```